frontend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mgm-interns/team-radio.git"
+  },
+  "engines": {
+    "node": "8.x"
+  },
+  "dependencies": {
+    "font-awesome": "^4.7.0",
+    "graphql": "^0.11.7",
+    "immutable": "^3.8.2",
+    "material-ui": "^1.0.0-beta.23",
+    "moment": "^2.19.4",
+    "node-sass-chokidar": "0.0.3",
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-flexbox-grid": "^2.0.0",
+    "react-redux": "^5.0.6",
+    "react-router-dom": "^4.2.2",
+    "react-scripts": "1.0.17",
+    "redux": "^3.7.2",
+    "subscriptions-transport-ws": "^0.9.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "devDependencies": {
```

Comment:
Hi @lybaokhanh, can you remove `lint-staged` and anything related to `eslint`? I will config them later in project root

Reply:
thanks! it's your turn @pvtri96